### PR TITLE
Remove the invisible format marks from the text chunks

### LINKED ISSUE #9186: PDF Search Issue: Not able to search with two words has space
Attach (recommended) or Link to PDF file here:
PDF is [Sample lease contract 26.pdf](https://github.com/mozilla/pdf.js/files/1500985/Sample.lease.contract.26.pdf)

Steps to reproduce the problem:
I am using [https://mozilla.github.io/pdf.js/web/viewer.html](url) to preview pdf by opening the pdf.

now am searching for "**made is**" but its not showing in the pdfjs

and while selecting and copying pdf  text it has no space in b/w words

### PR DESCRIPTION
**Pull request by calixteman**
- it aims to fix issue #9186.

### CODE REVIEW

File: test/test_manifest.json [hunk 1]
```diff
@@ -6230,5 +6230,13 @@
...
       "lastPage": 1,
       "type": "eq"
+   },
+   { "id": "issue9186",
+      "file": "pdfs/issue9186.pdf",
+      "md5": "d151857bb724ab9a291704a45a0b5d7f",
+      "rounds": 1,
+      "link": true,
+      "lastPage": 1,
+      "type": "load"
```
Comment (Snuffleupagus) [inline test/test_manifest.json:6240]: This should be `"type": "text"` instead, right?

File: test/unit/api_spec.js [hunk 2]
```diff
@@ -2147,6 +2147,25 @@ sozialökonomische Gerechtigkeit.`)
       await loadingTask.destroy();
     });
 
+    it("gets text content, with invisible text mark (issue 9186)", async function () {
```
Comment (Snuffleupagus) [inline test/unit/api_spec.js:2150]: Given that the test-case is linked, you'll need to add the following block to fix the GitHub Actions failure:
```javascript
if (isNodeJS) {
  pending("Linked test-cases are not supported in Node.js.");
}
```

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]

**Comment by Snuffleupagus** [comment]
The unit-test failures are already tracked elsewhere, so let's just check the integration-tests again.

/botio integrationtest

**Comment by calixteman** [comment]
What's the issue number for the scripting unit test failure (I can reproduce locally) ?

**Comment by Snuffleupagus** [comment]
> What's the issue number for the scripting unit test failure (I can reproduce locally) ?

Please see issue #14488 (it's too bad that GitHub hasn't got a proper "needinfo" functionality, besides the general @-ing a user).

/botio integrationtest

**Review by Snuffleupagus — APPROVED** [review]
r=me, since this looks reasonable.

(The intermittent integration-test failures on Windows, while unfortunate, should really be completely unrelated to this patch.)

**Comment by calixteman** [comment]
> r=me, since this looks reasonable.
> 
> (The intermittent integration-test failures on Windows, while unfortunate, should really be completely unrelated to this patch.)

Yeah I recently saw it too...

**Pull request merged**